packages/ui-kit/tailwind.config.js

```diff
@@ -0,0 +1,108 @@
+const getSpacing = (
+  base /* number */,
+  unit /* "px" | "rem" */,
+  values /* number[] */
+) => {
+  return values.reduce((acc, value) => {
+    return { ...acc, [value]: base * value + unit };
+  }, {});
+};
+
+const spacing = getSpacing(
+  0.4,
+  "rem",
+  [
+    0, 0.5, 1, 1.5, 2, 2.5, 3, 4, 5, 6, 6.5, 7, 8, 10, 11, 12, 14, 16, 18, 19,
+    21, 22, 28, 85, 256, 350,
+  ]
+);
+
+const theme = {
```

Comment:
We def need to use CSS variables here, please consult this with @bmigirl

Reply:
What we need essentially is a way to inject custom styling per entire project, not on single component level. This could also use some smart provider that injects some base styles based on the component,

Reply:
It will be done in separate PR.